src/core/pages/structuredMarkdown.test.tsx

```
@@ -7,9 +7,13 @@ import { render } from '@testing-library/react';
 import { structureMarkdownContent } from './structuredMarkdown';
 import { compiler } from 'markdown-to-jsx';
 
-function renderCompiledMarkdown(markdown: string) {
+function renderCompiledMarkdown(
+  markdown: string,
+  idPrefix: string = 'hdr',
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)